docs/architecture.mdx

```diff
@@ -0,0 +1,159 @@
+The blockchain data is maintained by a network of Tezos nodes.
+Nodes reach consensus on the next block before adding it to the chain.
+
+There are several Tezos blockchains.
+The primary Tezos network is referred to as Mainnet, and there are several other networks used for testing, referred to as testnets.
+Anyone can create new test networks if needed.
+
+
+The data inside a Tezos block includes operations such as transferring tokens between accounts, deploying or calling smart contracts, and performing governance operations.
+Blocks also include operations that are necessary for the management of the chain, including nodes' attestations that blocks are valid.
+For more information on the operations that can be included in blocks, see [Blocks and operations](https://tezos.gitlab.io/alpha/blocks_ops.html) in the Octez documentation.
+
+## Nodes
+
+A Tezos node has three main roles:
+
+- It validates blocks and operations
+- It broadcasts blocks and operations to other nodes and receives them from other nodes over a peer-to-peer network
+- It maintains a copy of the blockchain data and its associated state (also known as the ledger), which includes accounts and their balances, among other things
+
+Beside these technical roles, nodes must satisfy two other important requirements:
+
+- Support the governance of the blockchain
+- Ensure the extensibility of the blockchain with new clients of different kinds.
+
+In order to meet these requirements, the software that nodes run is structured according to two major principles:
+
+- It is separated into a shell and a protocol, to support protocol evolution.
+- It implements a client/server architecture, to allow composition with many other tools in a safe way.
+
+The Octez suite, which is an implementation of the Tezos node and other executables, instantiates these principles in the [Octez software architecture](https://tezos.gitlab.io/shell/the_big_picture.html).
+
+These two principles are detailed next.
+
+## Shell & Protocol
+
+The software that runs Tezos nodes is split into two main parts:
+
+- The shell, which handles low-level functions like data storage and peer-to-peer network communication
+- The economic protocol that interprets the block operations (e.g., transactions)
+
+The relationship between the shell and the protocol is like the relationship between an operating system and an application.
+The operating system stays stable while the application can update itself.
+In this way, Tezos can update how it works (its protocol) without requiring nodes to accept major changes to the software that they run (the shell).
+For example, nodes can update to a new protocol version without restarting the shell.
+
+### The Tezos self-amending protocol
+
+Unlike many other blockchains, Tezos is self-amending.
+Its nodes can update the protocol that controls the possible operations and how they are processed via an online governance process.
```

Comment:
If you don't like the comma before "via", let's separate in two sentences, to avoid the wrong reading "processed via an online governance process":
```suggestion
Its nodes can update the protocol that controls the possible operations and how they are processed; updates are performed via an online governance process.
```